docs/README.md

```diff
@@ -75,6 +87,38 @@ erDiagram
     string password "Passphrase to access the calendar"
     bool connected "Flag indicating if calendar is actively used"
     date connected_at "Date calendar was connected"
+    date time_created "UTC timestamp of calendar creation"
+    date time_updated "UTC timestamp of last calendar modification"
+  }
+  SUBSCRIBERS ||--o{ EXTERNAL_CONNECTIONS : create
+  EXTERNAL_CONNECTIONS {
+    int id PK "Unique connection key"
+    int owner_id FK "Person who creates and owns this connection"
+    string name "Custom connection title"
+    enum type "Connection type [zoom, google, fxa, caldav]"
+    string type_id "Universal unique identifier"
+    string token "Passphrase or passtoken for connecting"
+    date time_created "UTC timestamp of connection creation"
+    date time_updated "UTC timestamp of last connection modification"
+  }
+  SUBSCRIBERS ||--o{ INVITES : hold
+  INVITES {
+    int id PK "Unique invite key"
+    int subscriber_id FK "Person who holds this invite"
+    string code "Unique invitation code"
+    enum status "Invitation status [active, revoked]"
+    date time_created "UTC timestamp of invite creation"
+    date time_updated "UTC timestamp of last invite modification"
+    int owner_id FK "Person who creates and owns this invite"
```

Comment:
@MelissaAutumn It seems the field `owner_id` is duplicate to the field `subscriber_id` - can we remove one of them?

Reply:
I believe owner_id can be an admin. I'd have to recheck the code.

Reply:
Yep. We give users 10 invites they can use, and that's the owner_id (And this is optional for admin use.) Subscriber_id is the created user from an invite.

Reply:
Got it. Thanks!